README.md

```diff
@@ -70,6 +70,20 @@ and rubocop are installed locally, **not** just in a Docker container. Ensure
 `bundle install` installs the rubocop gem. Script adapted from
 [here](http://gmodarelli.com/2015/01/code_reviews_rubocop_pre_commit/).
 
+## Unit tests
+We use RSpec as our testing suite. Our tests for models are located under
+`spec/models`. Factories are provided by `FactoryBot` and are located under
+`spec/factories`.  To run them, go into your local Docker container with
+`docker exec -it <image_name> sh`, and run:
+```
+# Get your test database similar to your development database
+rake db:test:prepare
+
+# Run the actual tests
+rake spec
+```
+The last command can be `bundle exec rspec` or `rspec`.
```

Comment:
This is a bit confusing. What's the difference between `rake spec`/`bundle exec rspec`/`rspec`? If they're all the same, I think we can just put one of them.

Also maybe add in commands for running all test vs testing one specific class vs running one specific test in a class.
Eg.
```
# Run entire test suite
rspec spec

# To run only tests for one class do rspec {file_name}.
# For example,
rspec spec/models/position_spec.rb

# Run a specific test block for a class, do rsepc {file_name}:{line_number} where line_number is the first line the block starts on
rspec spec/models/position_spec.rb:17

Reply:
I like this. Probably should have one.